src/test/java/benchmark/BenchmarkUtils.java

```diff
@@ -1,11 +1,13 @@
 package benchmark;
 
-import com.google.common.io.Files;
-import graphql.Assert;
-
-import java.io.File;
+import java.io.BufferedReader;
```

Comment:
This file was changed in another PR due to be included in the forthcoming v22 release. We have to backport this too.